src/app/signup/signup.component.ts

```diff
@@ -40,7 +40,13 @@ export class SignupComponent implements OnInit {
     this.accountService.register(this.user)
       .subscribe(
         () => this.router.navigateByUrl('/login?registered=true'),
-        () => this.wrongCredentials = true,
+        (error) => {
+          if (error.error.errors) {
```

Comment:
Weird naming error.error.errors

Reply:
The first error is the default naming for that function. The other are fixed from the json result. So would be registrationError.error.errors better? I am not sure.

Reply:
I think it's fine how it is. It's just funny :)